include/dfu/suit_dfu.h

```diff
@@ -7,32 +7,31 @@
 #ifndef SUIT_DFU_H__
 #define SUIT_DFU_H__
 
-/**
+/** @file suit_dfu.h
+ *
  * @brief Module providing tools to the Local domain Core allowing to orchestrate the SUIT DFU.
- */
-
-/**
- * The following steps are required by the application to perform Device Firmware Update:
  *
- * (1) Call @ref suit_dfu_initialize to initialize the SUIT DFU module before any
- *     other SUIT functionalities are used. This function only has to be called once.
- *     This function must be called with at least APPLICATION init level (calling it
- *     in POST_KERNEL or earlier init level causes undefined behavior).
- * (2) Optional: If the SUIT manifest may require fetching of additional data in the
- *     suit-payload-fetch sequence, fetch sources should be registered using
- *     suit_fetch_source_register
- * (3) Store the SUIT candidate envelope in the non-volatile memory area defined as
- *     dfu_partition in the device tree.
- * (4) Optional, not supported yet: Store other needed data/detached payloads
- *     in dfu cache partitions.
- *     DFU cache pool 0 is not available at this stage (it is part of dfu_partition
- *     not occupied by the SUIT candidate envelope).
- * (5) When storing the SUIT candidate envelope is finished, call
- *     @ref suit_dfu_candidate_envelope_stored
- * (6) Call @ref suit_dfu_candidate_preprocess. If any fetch sources were registered
- *     in step (2) they might be used by this function to fetch additional data.
- * (7) Call @ref suit_dfu_update_start. This will start the firmware update by
- *     resetting the device and passing control over the update to the Secure Domain.
+ * The application requires the following steps to perform the Device Firmware Update:
+ *   1. Call @ref suit_dfu_initialize to initialize the SUIT DFU module before any
+ *      other SUIT functionality is used. This function only has to be called once.
+ *      This function must be called only once, with at least APPLICATION init level.
+ *      Calling this function in POST_KERNEL or earlier init level causes undefined behavior.
+ *   2. If the SUIT manifest requires the fetch of additional data in the suit-payload-fetch
```

Comment:
Like in the .rst file, I don't think the caller of this function can detect what the actual manifest requires. I would rephrase it to something like "If the design of the DFU system used by this device allows for the fetch of additional..."